### PR TITLE
Portfolio Setup and Mobile-first: Work section and footer

### PR DESCRIPTION
**Pull request by SamTush**
Portfolio setup and mobile-first: Add work section and footer

- Add work section.
- Add footer and form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,8 +47,189 @@ <h1 class="h1">I'm Samson<br>Glad to see<br>you!</h1>
...
+                    </div>
+                    <div>
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                    </div>
+                    <div class="small-btn-div">
+                        <div class="small-btn"><a>html</a></div>
+                        <div class="small-btn"><a>css</a></div>
+                        <div class="small-btn"><a>javaScript</a></div>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle: The  [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) states that  you should  use ul and li for categories in "Work" section. Html, Css, and Javascript should be list items.
Screenshot from 2023-01-14 21-17-09

File: index.html [hunk 2]
```diff
@@ -47,8 +47,189 @@ <h1 class="h1">I'm Samson<br>Glad to see<br>you!</h1>
...
+                    </div>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle: The  [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) states that  you should  use ul and li for categories in "Work" section. Html, Css, and Javascript should be list items.
Screenshot from 2023-01-14 21-17-09

File: index.html [hunk 3]
```diff
@@ -47,8 +47,189 @@ <h1 class="h1">I'm Samson<br>Glad to see<br>you!</h1>
...
```
Comment (bienvenuushindi) [inline index.html]: :red_circle: The  [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) states that  you should  use ul and li for categories in "Work" section. Html, Css, and Javascript should be list items.
Screenshot from 2023-01-14 21-17-09

File: index.html [hunk 4]
```diff
@@ -47,8 +47,189 @@ <h1 class="h1">I'm Samson<br>Glad to see<br>you!</h1>
...
```
Comment (bienvenuushindi) [inline index.html]: :red_circle: The  [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) states that  you should  use ul and li for categories in "Work" section. Html, Css, and Javascript should be list items.
Screenshot from 2023-01-14 21-17-09

File: style.css [hunk 5]
```diff
@@ -103,3 +116,278 @@ p {
...
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+button {
```
Comment (bienvenuushindi) [inline style.css:239]: :red_circle:  Good job creating and styling all the buttons inside your project :100:, but as the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) states, you should add interactions like Active, Hover and Pressed.
buttond

File: index.html [hunk 6]
```diff
@@ -47,8 +47,189 @@ <h1 class="h1">I'm Samson<br>Glad to see<br>you!</h1>
...
+                <div class="frame">
+                    <div class="framework">
+                        <a>Frameworks</a>
+                        <img src="images/Icon-arrow.png" alt="Icon-arrow">
+                    </div>
+                    <hr>
+                    <div class="skills">
+                        <a>Skills</a>
+                        <img src="images/Icon-arrow.png" alt="Icon-arrow">
+                    </div>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle: The  [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) states that  you should  use ul and li for categories in "Work" section. Html, Css, and Javascript should be list items.
Screenshot from 2023-01-14 22-56-53

File: style.css [hunk 7]
```diff
@@ -103,3 +116,278 @@ p {
...
+  background-color: #ebebff;
+  padding: 0 24px;
+}
+
+.grid-container {
+  display: grid;
+  grid-auto-rows: minmax(552px, auto);
+}
+
+.grid-item {
```
Comment (bienvenuushindi) [inline style.css:133]: :red_circle:The  [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) states that you should lay out the cards elements of this work section using CSS FlexBox. Kindly Use CSS FlexBox for the cards elements. Kudos for using  CSS Grid to lay out an the wrok section container  :+1: :100: .
Screenshot from 2023-01-14 21-08-06

File: index.html [hunk 8]
```diff
@@ -47,8 +47,189 @@ <h1 class="h1">I'm Samson<br>Glad to see<br>you!</h1>
...
+
+    <section class="about">
+        <div class="grid-container">
+            <div class="footer-text">
+                <h2>About<br>Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div class="con">
+                    <a class="connect">
+                      LET'S CONNECT
+                    </a>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle: I think the right semantic tag here is hX tag (heading) than a tag link. A heading is a short phrase that indicates what the next section is about.

File: index.html [hunk 9]
```diff
@@ -47,8 +47,189 @@ <h1 class="h1">I'm Samson<br>Glad to see<br>you!</h1>
...
+                </div>
+                <ul class="social">
+                    <li><a><img src="images/twitter.svg" alt="twitter" class="icon"></a></li>
+                    <li><a><img src="images/Linkedin.svg" alt="Linkedin" class="icon"></a></li>
+                    <li><a><img src="images/Vector-3.svg" alt="Vector-3" class="icon"></a></li>
+                    <li><a><img src="images/Vector-2.svg" alt="Vector-2" class="icon"></a></li>
+                    <li><a><img src="images/Vector-4.svg" alt="Vector-4" class="icon"></a></li>
+                </ul>
+                <button><a>Get my resume</a></button>
```
Comment (bienvenuushindi) [inline index.html]: :red_circle: I think it is a bad practice to nest a tag inside a button tag. Buttons are supposed to perform a specific action like submitting a form. When you nest one inside the other, it makes it confusing as to what action you want performed. Kindly remove keep the text and remove the a tag inside the button

File: index.html [hunk 10]
```diff
@@ -47,8 +47,189 @@ <h1 class="h1">I'm Samson<br>Glad to see<br>you!</h1>
...
+            <div class="text-contact">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+                </p>
+                <div class="form-div">
+                    <form action="/action_page.php">
+                      <input type="text" id="fname" name="firstname" placeholder="Yeremias">
+                      <input type="text" id="lname" name="lastname" placeholder="natajayanj14@gmail.com">
+                      <textarea name="text" id="text" cols="30" rows="10"></textarea>
+                      <button><a>Get In Touch</a></button>
```
Comment (bienvenuushindi) [inline index.html:226]: :orange_circle: [OPTIONAL] I think it is a bad practice to nest a tag inside a button tag. Buttons are supposed to perform a specific action like submitting a form. When you nest one inside the other, it makes it confusing as to what action you want performed. Kindly remove keep the text and remove the a tag inside the button

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @SamTush ,

Good job imeplementing most of the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔


## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- [ ] Please check the inline comments below.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @SamTush , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Green commits
✅ ... Great readme file

---------
### Optional suggestions



- [ ] Take a look at this record and make the necessary changes to match the original design, this comment is marked as optional but I would suggest you take time to work and fix this issue

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**